README.md

```diff
@@ -68,7 +64,7 @@ You can also use `setHeaders()` and `addRow` with Arrays.
 
 ## Example 3: Bordered Table with Left Margin Width 4
 
-    require_once 'ConsoleTable.php';
+    require "vendor/autoload.php";
```

Comment:
Can you please remove this line since it is only valid for composer users? We shall not include `require` in all examples.